src/com/laba/discountpackage/Discount.java

```diff
@@ -0,0 +1,39 @@
+package com.laba.discountpackage;
+
+
+public class Discount implements DiscountInterface {
+
+    private static Boolean newUser= true;
+    private int discountId;
+    private double discountCode;
+
+    public Discount(int discountId,Boolean newUser) {
+        this.discountId = discountId;
+        this.newUser = newUser;
+    }
+
+    public final double discountTotalPrice(double totalPrice,Boolean newUser) {
+        if(newUser == true) {
+            discountCode = .85;
+            return this.newUser(totalPrice);
+        }
+        else if(totalPrice>50) {
+            discountCode = .90;
+            return this.freeShipping(totalPrice);
+        }
+        return totalPrice;
```

Comment:
```suggestion
        if (newUser == true) {
            discountCode = .85;
            return this.newUser(totalPrice);
        } else if (totalPrice > 50) {
            discountCode = .90;
            return this.freeShipping(totalPrice);
        }
        return totalPrice;
```
Doublecheck the doc with formatting and format your code, i will not ping the same issue again, try to find all places on your own.